README.md

```diff
@@ -342,10 +342,20 @@
 
   - Founder: [@mrmakoko](https://twitter.com/mrmakoko)
 
-  - Industry: Venture Capital
+
```

Comment:
Please also include this information in the `techCompanies.js`

Reply:
Done that already in #125

Reply:
Okay thanks. I'll review it now